pytest_mh/utils/firewall.py

```diff
@@ -289,95 +302,85 @@ def _next_priority(self) -> int:
         self._priority -= 1
         return priority
 
-    def add_direct_rule(
+    def add_policy(
         self,
-        chain: str,
-        args: list[Any],
+        name: str,
         *,
-        table: str = "filter",
-        ip_family: Literal["ipv4", "ipv6", "all"] = "all",
+        ingress: str | None = None,
+        egress: str | None = None,
         priority: int | None = None,
+        target: Literal["CONTINUE", "ACCEPT", "DROP", "REJECT"] | None = None,
     ) -> int:
         """
-        Add a new direct rule.
+        Add a new (permanent) policy.
```

Comment:
But it is still removed in teardown, right?

Reply:
I see your point. It's done now.